sidebarsValidate.js

```diff
@@ -1,140 +1,136 @@
 module.exports = {
   docs: [
     {
-      type: 'category',
-      label: 'Overview',
+      type: "category",
+      label: "Overview",
       collapsed: false,
-      items: [
-        'index'
-      ],
+      items: ["index"],
     },
     {
-      type: 'category',
-      label: 'Node Concepts',
+      type: "category",
+      label: "Node Concepts",
       collapsed: false,
       items: [
-        'validator/get_started',
-        'validator/specs',
-        'validator/providers',
-        'validator/vpsExplained',
+        "validator/get_started",
```

Comment:
Did your IDE make these changes? Can you update this PR without the 'minor' changes like quotes changing so the diff here is not substantial?

Reply:
Reverted the auto formatting.